### ISSUE TITLE
Allow functionality to open two tabs side by side.

### ISSUE THREAD

**Issue by sujayvenaik** [a85/POSTMan-Chrome-Extension#9180]
**Is your feature request related to a problem? Please describe.**
A lot of time i want to compare two requests/reponses from apis. I always have to switch tabs to see.

**Describe the solution you'd like**
if we could allow two request/examples tabs side by side.

**Describe alternatives you've considered**
1. open two windows
2. open tabs, and keep switching between them.

**Additional context**

**Comment by ahaq0** [comment]
Would be a great addition

**Comment by charlesritchea** [comment]
My team uses postman extensively, but I often use JUnit test drivers with our Kotlin client, switching from IntelliJ to Postman is always jarring and this is one of the main reasons. You can't even have multiple tabs of variables open let alone side-by-side

**Comment by jahumphr** [comment]
It's end of 2022 and surprised this feature still doesn't exist.

**Comment by jatinchaudhary** [comment]
Its 2023, come on postman team. 
At least release in beta version

**Comment by christin3** [comment]
Side by side would be so helpful!!

**Comment by Ulkryxx** [comment]
I'm currently logged into postman web browser side by side the app comparing files like a barbarian. I hope this gets some attention soon.

**Comment by jaafari-R** [comment]
One day... one day it will happen.
May that day be close :smiley:

**Comment by TamirYaariSpot** [comment]
Bring that day closer postman team!

**Comment by rohitsasi720** [comment]
Still waiting for THE day :")

**Comment by MauriJrP** [comment]
Starting to wonder if this is not just another impossible dream 😞

**Comment by druryyl** [comment]
> I'm currently logged into postman web browser side by side the app comparing files like a barbarian. I hope this gets some attention soon.

I do the same thing. But the browser version cann't access localhost. Need split tab feature so badly...

**Comment by domjancik** [comment]
A request and environment open side by side would also be a great use case for this when modifying environment variables.

Please prioritize this! ❤️

**Comment by jamescze** [comment]
Please, add this as soon as possible. It is really basic functionality, which should have been existing since begging.

Thank you. :)

**Comment by haoxi911** [comment]
It happens a lot to us where I need to copy response from a tab and use as the request body in another tab, being able to see both tabs at the same time helps me identifying any typo or missing fields. So please prioritize this! ❤️

**Comment by talitagomesfe** [comment]
I'm currently comparing two (apparently) equal requests where one fails and the other succeeds, trying to figure out what is different between them and having to switch tabs is so slow 😢

Having side by side requests would be so helpful!

**Comment by jamescze** [comment]
Workaround: Install official VS Code extension and use Postman there. There is possible to have two  (or more) opened tabs side by side :)

https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode

**Comment by Jeldik** [comment]
Vote for!

**Comment by domjancik** [comment]
> Workaround: Install official VS Code extension and use Postman there. There is possible to have two (or more) opened tabs side by side :)
> 
> https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode

Actually, I've tried it and the Postman tab itself seems to be limited to a single instance. Ie. opening a different request or env view will always open in the same tab and there is no drag and drop to new panels or other way to split it up as far as I can see.

Splitting with other views such as regular code files, markdown or anything unrelated to Postman of course still works and may be useful.

In general I like the extension, thanks for the recommendation, but it does not solve the Postman split view issue as discussed here.

**Comment by abhijeetborole** [comment]
@domjancik Could you share the version you're on? You can open different tabs side by side in the VSCode extension. Are you trying to open the same request/env side by side or different ones?

https://github.com/postmanlabs/postman-app-support/assets/9638586/917e6e17-d347-4102-aaae-378768495d3b

**Comment by domjancik** [comment]
Ah, you're right it is there. I had to press on the 3 dot menu and select `Enable tabs`.

It was not enabled by default, making everything open in a single one.  Extension version 0.5.4

**Comment by tonynecula** [comment]
October 2023 and still not able to split screen! I'd pay to be able to do it

**Comment by Farfeght** [comment]
January 2024. I would still love this feature.

**Comment by sislam-provenir** [comment]
I can't believe this feature is not available, it would really take the usability to another level.

**Comment by rohansanjay** [comment]
yes please!

**Comment by j-peace** [comment]
Same here ++

**Comment by markwasilwa** [comment]
I needed this today, Postman pls :-)

**Comment by charlesritchea** [comment]
@markwasilwa not ideal, but as a workaround you can open a new Postman Window with ctrl-shift-n and put them side-by-side. It kind of feels like this is a shortcoming of the entire Electron based (or influenced) app ecosystem. They would have loved Windows 3 era :laughing:

**Comment by ghost** [comment]
> @markwasilwa not ideal, but as a workaround you can open a new Postman Window with ctrl-shift-n and put them side-by-side. It kind of feels like this is a shortcoming of the entire Electron based (or influenced) app ecosystem. They would have loved Windows 3 era 😆

true, it's not ideal. If only 2 Postman Windows side-by-side could compare the request/response with "diffing". 
I needed to compare two large JSON payloads to the same endpoint, I used an IDE

**Comment by Politta** [comment]
Using on VSCode worked well for me as a workaround. Ideally this would be part of the postman app tho!

**Comment by imakecodes** [comment]
Hey @postman-staff what about this? WE NEED THIS!

**Comment by AWahba2** [comment]
Worked with me using VSCode Extension.

**Comment by 9yo** [comment]
huge plus! add please:)

**Comment by northfacejmb** [comment]
really was wanting for this today =(

**Comment by jorgecabanasrealcore** [comment]
+1

**Comment by ArCiGo** [comment]
Is there any update on this?

**Comment by GwFreak01** [comment]
+1 support on this pls!

**Comment by phil-sola** [comment]
Please postman add this feature!! This would be so helpful 🙏

**Comment by akshtrikha** [comment]
CFBR :')

**Comment by snsc-tset** [comment]
This would be an extremely good feature for Postman.

**Comment by mschinis** [comment]
would love to have this, especially when working on larger screens

**Comment by johnnyoshika** [comment]
Same here

**Comment by rogergazeta** [comment]
up ;c

**Comment by mmendoncaAzion** [comment]
2025 and still waiting

**Comment by Spyridonidis98** [comment]
july 2025 still nothing

**Comment by fahdfady** [comment]
how is this not a feature already?

**Comment by vincx2000** [comment]
just came back from vacation and still no features :(

**Comment by LasseMunk** [comment]
up

**Comment by PSPanishetti** [comment]
Really ? I thought I was the first guy to think of this feature ... Come On Postman team give us this feature

**Comment by ashevchenkosps** [comment]
up

**Comment by Bishoy334** [comment]
+1

**Comment by Catherine-Giugno-Wegmans** [comment]
+1 This would be really cool! Just tried to search the feature and ended up here haha

**Comment by MikaelPorttila** [comment]
Split panes/tabs would be a great addition, but also a baseline for good DX in this day and age
